package.json

```diff
@@ -88,12 +89,15 @@
     "graceful-fs": "^4.1.11",
     "invariant": "^2.2.0",
     "istanbul": "^0.4.5",
+    "jest": "^21.2.1",
     "js-yaml": "^3.6.1",
     "jsdom": "^9.2.1",
     "kcheck": "^2.0.0",
     "madge": "^1.6.0",
     "minimist": "^1.2.0",
     "prettier": "1.5.2",
+    "react": "^16.0.0",
+    "react-test-renderer": "^16.0.0",
```

Comment:
It might be useful to land a separate PR for the test infra changes and some basic test that you then update in the follow up since the rest of this might need more review and by different people.

Reply:
I was hesitant to remove the tests from the code – that did cross my mind though. I still feel they're relevant here though, I'm not sure how a basic test using the current infrastructure would warrant any real gain.

Reply:
You can keep the tests in this PR but in the infra PR you can just have a single test that does `it('succeeds', () => {})` and just checks in jest and such. If there are controversies about using jest or the yarn file update, we can keep it to that PR. But it'll also help review this one since it'll be fewer files to think about once the infra one lands.

Reply:
You can still stack this on top of the other.

Reply:
So move `test-react.js` out of this PR? Sorry, I'm confused :/